src/org/zaproxy/zap/extension/simpleExample/resources/Messages.properties

```diff
@@ -2,11 +2,11 @@
 #
 # This file defines the default (English) variants of all of the internationalised messages
 
-simpleExample.desc					= A Simple Example Add-on
-simpleExample.error.nofile			= File not found : {0}
-simpleExample.panel.msg				= <html>This is a simple example of a status panel</html>
-simpleExample.panel.title			= Simple Example
-simpleExample.popup.msg				= Example right click menu invoked for:\n {0}
-simpleExample.popup.title			= Simple Example: Example right click menu
-simpleExample.topmenu.tools.msg		= Simple Example: Example message  
+simpleExample.desc								= A Simple Example Add-on
+simpleExample.error.nofile				= File not found : {0}
+simpleExample.panel.msg						= <html>This is a simple example of a status panel</html>
+simpleExample.panel.title					= Simple Example
+simpleExample.popup.msg						= Example right click menu invoked for:\n {0}
+simpleExample.popup.title					= Simple Example: Example right click menu
+simpleExample.topmenu.tools.msg		= Simple Example: Example message
```

Comment:
All the spacing before the equals signs could go away (IMHO), it's totally unnecessary.

(<sub>Ref: https://github.com/zaproxy/zaproxy/issues/276</sub>)